README.rst

```diff
@@ -48,6 +48,15 @@ Environment variables available to run and test the job:
     Prescriptions repository URL.
 - **THOTH_PRESCRIPTIONS_REFRESH_SUBCOMMAND**
     Command options for prescriptions-refresh-job CLI.
+- **THOTH_USER_API_HOST**
+    Thoth USER_API host.
+- **THOTH_PRESCRIPTIONS_REFRESH_ML_PACKAGES**
+    Representative packages for ML.
+- **THOTH_PRESCRIPTIONS_REFRESH_NLP_PACKAGES**
+    Representative packages for NLP.
+- **THOTH_PRESCRIPTIONS_REFRESH_CV_PACKAGES**
+    Representative packages for CV.
+
```

Comment:
The file has just been reformatted (#107) so this needs rebasing

Reply:
Done ! 🚀